opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -261,10 +263,39 @@ class TraceState(typing.Dict[str, str]):
         https://www.w3.org/TR/trace-context/#tracestate-field
     """
 
+    MAX_TRACESTATE_VALUES = 32
```

Comment:
Do these really belong here? I think TraceState is not tightly bound to the W3C trace state.

Reply:
removed this logic entirely.